modules/openxBidAdapter.js

```diff
@@ -1,18 +1,30 @@
 const bidfactory = require('src/bidfactory.js');
 const bidmanager = require('src/bidmanager.js');
 const adloader = require('src/adloader');
+const ajax = require('src/ajax');
 const CONSTANTS = require('src/constants.json');
 const utils = require('src/utils.js');
 const adaptermanager = require('src/adaptermanager');
 
 const OpenxAdapter = function OpenxAdapter() {
   const BIDDER_CODE = 'openx';
   const BIDDER_CONFIG = 'hb_pb';
+  const BIDDER_VERSION = '1.0.1';
   let startTime;
+  let timeout = $$PREBID_GLOBAL$$.bidderTimeout;
```

Comment:
Regarding the timeout, there is at least one case which is not captured. If the old style PREBID_TIMEOUT variable is used in the style 

`    var PREBID_TIMEOUT = 700;`
`    // Load GPT when timeout is reached.`
`    setTimeout(initAdserver, PREBID_TIMEOUT);`

like it's done in the [example](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/pbjs_example_gpt.html)  the bidmanager will have no clue about it and default to 3000 instead. 

Therefore, I'd recommend 
`bt: Math.min($$PREBID_GLOBAL$$.cbTimeout || $$PREBID_GLOBAL$$.bidderTimeout, window.PREBID_TIMEOUT || 0)` 
instead